cpu/sam0_common/periph/rtc.c

```diff
@@ -98,6 +98,8 @@ void rtc_init(void)
 {
     _rtc_clock_setup();
     rtc_poweron();
+    /* WORKAROUND: prevent from being stuck in CLOCK SYNCBUSY if RTC disabled */
```

Comment:
Is this specific to samr34 or all saml21 are affected ?

Reply:
IIRC I only encountered this case for SAMR34

Reply:
so maybe add:
```c
#ifndef CPU_MODEL_SAMR34J18B
/* WORKAROUND: prevent from being stuck in CLOCK SYNCBUSY if RTC disabled */
RTC->MODE2.CTRLA.bit.CLOCKSYNC = 0;
#endif
```

Reply:
if available, please add a reference to the datasheet...